### PR TITLE
tabs with router by Yevhen Sahach

### PR DESCRIPTION
**Pull request by eugeneSahach**
https://yevhen-sahach.github.io/react_tabs-with-router/

### CODE REVIEW

**Comment by irenhh** [comment]
fix this part:
```After page reload you should see the content of a tab matching the id from the URL```

File: src/components/TabsContent/TabsContent.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TabsContent = ({ tabs, currentTabId }) => (
+  <>
```
Comment (YuriiHoliuk) [inline src/components/TabsContent/TabsContent.js]: unneeded

File: src/components/TabsContent/TabsContent.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <p>{tabs.find(tab => tab.id === currentTabId).content}</p>
```
Comment (YuriiHoliuk) [inline src/components/TabsContent/TabsContent.js]: better to pass content to this component
TabsContent shouldn't receive all tabs

File: src/components/TabsPage/TabsPage.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import TabsContent from '../TabsContent/TabsContent';
+
+const TabsPage = ({ match, tabs }) => (
```
Comment (YuriiHoliuk) [inline src/components/TabsPage/TabsPage.js]: do you really need all mach as prop here?

File: src/App.css [hunk 4]
```diff
@@ -1 +1,16 @@
...
+  border-bottom: none;
+}
 
+.link {
+  text-decoration: none;
+  color: #000;
+  margin-bottom: -1px;
+}
+
+.border-bottom {
```
Comment (YuriiHoliuk) [inline src/App.css]: css classes shouldn't be named as styles inside

Comment (yevhen-sahach) [reply src/App.css]: even if I decided to use atomic methodology of naming?

**Pull request closed without merging**